feel/feeldiscr/operatorinterpolation.hpp

```diff
@@ -1074,30 +1258,26 @@ OperatorInterpolation<DomainSpaceType, ImageSpaceType,IteratorRange,InterpType>:
 
                                 for ( uint16_type jloc = 0; jloc < domain_basis_type::nLocalDof; ++jloc )
                                 {
-                                    uint16_type compDomain = (domain_basis_type::is_product)? comp : 0;
+                                    uint16_type nCompDomain = (domain_basis_type::is_product)?domain_basis_type::nComponents:1;
+                                    for(int cdomain = 0;cdomain < nCompDomain;++cdomain )
+                                    {
```

Comment:
why is it a bug ? for the id() expression we should have only a block diagonal matrix no ?
the issue is that we have more computation because all components are considered coupled.
The tests are still passing although they are slower (which can be explained by the coupling).